morph/utils.js

```diff
@@ -632,15 +634,22 @@ export function replacePropWithDataValue(value, dataGroup) {
   } else if (dataGroup.data[0].isConstant) {
     if (propValue === 'value') {
       return dataGroup.data[0].name
+    } else if (propValue === '!value') {
+      return `!${dataGroup.data[0].name}`
     } else {
       throw new Error(
         `Property ${propValue} is not available on constant data, only "value" is a valid option`
       )
     }
   } else {
     if (propValue === 'isInvalid') return `!${dataGroup.variables['isValid']}`
+    if (propValue === '!isInvalid') return `${dataGroup.variables['isValid']}`
```

Comment:
lol shouldn't we refactor the app's code instead? :P

Reply:
yes, most likely. I thought maybe we should take this case into account, just in case someone has these crazy ideas (they should really use the positive version here indeed)

Reply:
yeah lol, well, it doesn't hurt anyone to support it tbh thanks!